src/Tribe/Ticket_Object.php

```diff
@@ -507,11 +507,20 @@ public function is_in_stock() {
 				return true;
 			}
 
-			$remaining = $this->inventory();
+			/* @var Tribe__Cache $cache */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: 77985a88048f8626ef3a011374f088df</sub>